sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -579,7 +579,6 @@ async def test_mgmt_queue_async_update_dict_success(self, servicebus_namespace_c
         finally:
             await mgmt_service.delete_queue(queue_name)
     
-    @pytest.mark.liveTest
```

Comment:
out of curiosity, do you know how annotation work? and by removing this, the livetest pipeline triggered by PR will not run mgmt tests in live mode while nightly running still runs in live mode?

Reply:
🤔 hmm, I'm not quite sure. I'll do some more investigation and get back to you!